packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/TestUtils.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_datastore
```

Comment:
I would avoid `TestUtils` as a file name. Violates the SRP. In other projects I have called this utility as `TestResources`, whose single responsibility is to read content from the test resources directory, only.

Reply:
Makes sense. how about `TestResourcesReadUtil`?